src/main/java/org/sagebionetworks/bridge/exporter/record/ExportType.java

```diff
@@ -0,0 +1,28 @@
+package org.sagebionetworks.bridge.exporter.record;
+
+import org.joda.time.DateTime;
+
+public enum ExportType {
```

Comment:
This class always assumes that DAILY endDateTime will be on the same day, but HOURLY endDateTime will be on the next hour. Which is it? If it's not consistent, this means we'll either be losing a millisecond on the DAILY case, or we have an overlapping millisecond on the HOURLY case.